mewarpx/tests/test_direct_solver.py

```diff
@@ -38,33 +38,34 @@ def test_direct_solver():
     run = diode_setup.DiodeRun_V1(
         dim=dim,
         DIRECT_SOLVER=DIRECT_SOLVER,
-        V_ANODE_CATHODE=VOLTAGE,
         V_ANODE_EXPRESSION=(
-            lambda t: VOLTAGE * np.sin(2.0 * np.pi * FREQ * t) if DIRECT_SOLVER
-            else "%.1f*sin(2*pi*%.5e*t)" % (VOLTAGE, FREQ)
+            (lambda t: VOLTAGE * np.sin(2.0 * np.pi * FREQ * t))
+            if DIRECT_SOLVER else f"{VOLTAGE}*sin(2*pi*{FREQ:.5e}*t)"
```

Comment:
Once the parser is available from Python, this will be able to go away, right?

Reply:
Yeap, that is the hope / plan.